### DISCUSSION TITLE
Federated timeline not working, cannot follow myself from other instance

### DISCUSSION THREAD

**Discussion by akidinatophat** [mastodon/mastodon#22891] [Q&A] [answered]
Hi, I recently setup an instance of version 4.0.2 at https://nortelnetworks.dev , and I cannot see any posts from other instances on my Federated timeline. Also, when I try searching my profile on another instance, it just says "Could not find anything for these search terms
". Am I doing something wrong?

**Comment by StrangeMinds1960** [comment]
I'm relatively new to this myself, but I'll try to give a few pointers

How long has your instance been running?  Federation takes time
Federation also depends (I think) on following people and having followers
You probable know this already, but the format for a username is @somebody@instance.tld
When you set up the instance, did you opt for a federated instance or a private one?
Do any instances show on https://nortelnetworks.dev/admin/instances (watch that 'MODERATION' selector at the top, should be on 'ALL')

**Reply by akidinatophat** [reply]
I set it up yesterday so probably 12+ hours I think..

If you're talking about single-user mode, it's not single user mode.

The moderation selector is on ALL, but it just says no domains found, even when I try to search for a domain like mstdn.social.

**Comment by akidinatophat** [comment, marked as answer]
solved, apparently it was a cloudflare proxying issue, weird. but i set it to dns only it works fine. thanks!
